protocol/0056-REWA-rewards_overview.md

```diff
@@ -1,73 +1,120 @@
+
+Note that the market settlement asset has nothing to do in particular with the asset used to pay out a reward for a market. 
+That is, a participant might recieve rewards in the settlement asset of the market, in VEGA governance tokens, and in any number of other unrelated tokens (perhaps governance of "loyalty"/reward tokens issued by LPs or market creators, or stablecoins like DAI).
+
+Reward accounts can be individually funded by normal single transfers. They can also be funded on an ongoing (recurring) basis in groups, pro-rata by their relative total reward metrics for the given reward type. See [transfers](./0057-TRAN-transfers.md).
+
+Reward accounts and balances must be saved in [LNL checkpoints](./0073-LIMN-limited_network_life.md) to ensure all funds remain accounted for accross a restart.
 
-Reward accounts and balances are to be saved in [LNL checkpoint](./0073-LIMN-limited_network_life.md). 
 
 ## Reward distribution
 
 All rewards are paid out at the end of any epoch *after* [recurring transfers](0057-TRAN-Transfers.md) have been executed. 
-There are no fractional payouts and no delays.
+The entire reward account balance is paid out every epoch unless the total value of the metric over all parties is zero (there are no fractional payouts). 
+There are no payout delays, rewards are paid out instantly at epoch end.
 
-### For fee based metrics
+Rewards will be distributed pro-rata by the party's reward metric value to all parties that have metric values `>0`. That is, if we have reward account balance `R` and parties `p_1 – p_n` with non-zero metrics on the market in question:
 
-Every epoch the entire reward account for every [market in scope, metric type, payout asset] will be distributed pro-rata to the parties that have the metric `>0`. 
-That is if we have reward account balance `R`
 ```
 [p_1,m_1]
 [p_2,m_2]
 ...
 [p_n,m_n]
 ```
-then calculate `M:= m_1+m_2+...+m_n` and transfer `R x m_i / M` to party `p_i` at the end of each epoch. 
-If `M = 0` (no-one incurred / received fees for a given reward account)  then nothing is paid out of the reward account and the balance rolls into next epoch. 
-
-Metrics are reset at the end of the epoch, so in the case where there is no reward account or no reward balance at the end of the epoch for the given market in scope, the participants contributing to the relevant metric will not be compensated for their contribution. Their contribution to fees is not being carried over to the next epoch. 
 
-Metrics will be calculated using the [decimal precision of the settlement asset](0070-MKTD-market-decimal-places.md).
+Then calculate `M:= m_1+m_2+...+m_n` and transfer `R x m_i / M` to party `p_i` (for each `p_i`) at the end of the epoch.
 
-### For market creation metrics
+If `M = 0` (no-one incurred or received fees as specified by the metric type for a given *market scope*) then nothing is paid out of the reward account for that *market scope* and the balance rolls into next epoch. 
```

Comment:
if M = 0 the reward account would not have been funded because the transfer would not have happened.

Reply:
updated